tests/models/whisper/test_modeling_tf_whisper.py

```diff
@@ -650,13 +650,15 @@ def _test_large_logits_librispeech(in_queue, out_queue, timeout):
         input_speech = _load_datasamples(1)
 
         processor = WhisperProcessor.from_pretrained("openai/whisper-large")
-        processed_inputs = processor(audio=input_speech, text="This part of the speech", return_tensors="tf")
+        processed_inputs = processor(
+            audio=input_speech, text="This part of the speech", add_special_tokens=False, return_tensors="tf"
+        )
         input_features = processed_inputs.input_features
-        labels = processed_inputs.labels
+        decoder_input_ids = processed_inputs.labels
```

Comment:
I remembered in a previous commit, you need to use a hard-coded tensor, which is the labels with some tokens removed.

Here the name `decoder_input_ids` is good, but the value is just `processed_inputs.labels`.

I am somehow confused by if we should have some tokens removed from labels or not, or it doesn't matter at all, and both work? (I will be shocked if this is the case)

Reply:
Haha feel free to read the suggestions add by @sanchit-gandhi but basically, it gives the same result. I hard coded to remove the `special_tokens`, and passing `add_special_tokens=False` is enough to remove the special tokens that where failing the tests.

Reply:
Yes, I understand now with all the explanation :-) Thanks.